test/common/access_log/access_log_formatter_test.cc

```diff
@@ -106,7 +107,7 @@ TEST(AccessLogFormatterTest, plainStringFormatter) {
 
   EXPECT_EQ("plain", formatter.format(header, header, header, stream_info));
   EXPECT_THAT(formatter.formatValue(header, header, header, stream_info),
-              ProtoEq(stringValue("plain")));
+              ProtoEq(ValueUtil::stringValue("plain")));
```

Comment:
Near the top of this file (e.g. lines 33-52 or so), there are test-private implementations of these new ValueUtil functions that can be deleted.

Reply:
726f7e3